### PR TITLE
Added musl base flag. Alpine doesn't support backtrace

### PR DESCRIPTION
**Pull request by adubovikov**

### CODE REVIEW

**Comment by seladb** [comment]
Thanks @adubovikov for your contribution!
I think a more standard way to support musl-based platforms would be to create a new `mk/platform.mk.musl` that adds `-DMUSL` to `GLOBAL_FLAGS`:

```makefile
# musl specific build flags
GLOBAL_FLAGS := -DMUSL
```

And then in `configure-linux.sh` just append the content of this file, same as we do with macOS M1:
https://github.com/seladb/PcapPlusPlus/blob/master/configure-mac_os_x.sh#L143-L148

**Comment by seladb** [comment]
@adubovikov did you have a chance to look at my previous comment?

**Comment by adubovikov** [comment]
@seladb yes, sorry, I was out these days and will do as you suggested.

**Comment by adubovikov** [comment]
now should look better

File: .gitignore [hunk 1]
```diff
@@ -38,6 +38,8 @@ mk/PcapPlusPlus.mk
 mk/platform.mk
 mk/PcapPlusPlus.mk-e
 3rdParty/LightPcapNg/zstd.mk
+3rdParty/MemPlumber/musl.mk
```
Comment (seladb) [inline .gitignore:41]: I don't think this is needed, is it?

File: 3rdParty/MemPlumber/Makefile [hunk 2]
```diff
@@ -8,7 +8,7 @@ include Makefile.defs
 
 %.o: %.cpp
 	@echo Building MemPlumber file: $<
-	@$(CC) $(GLOBAL_FLAGS) -Wall -O2 -g -c -o "$@" "$<"
+	@$(CC) $(GLOBAL_FLAGS) $(DEFS) -Wall -O2 -g -c -o "$@" "$<"
```
Comment (seladb) [inline 3rdParty/MemPlumber/Makefile:11]: do we need to add `$(DEFS)` here?

File: .gitignore (hunk 1 above)
Comment (adubovikov) [reply .gitignore:41]: sorry, yes, let me remove that

File: 3rdParty/MemPlumber/Makefile (hunk 2 above)
Comment (adubovikov) [reply 3rdParty/MemPlumber/Makefile]: fixed.

**Review by seladb — APPROVED** [review]

**Pull request merged**